lib/bolt/node/winrm.rb

```diff
@@ -337,7 +518,23 @@ def _run_task(task, input_method, arguments)
           # NOTE: cannot redirect STDIN to a .ps1 script inside of PowerShell
           # must create new powershell.exe process like other interpreters
           # fortunately, using PS with stdin input_method should never happen
-          output = execute("try { &""#{remote_path}"" } catch { exit 1 }")
+          output = execute(<<-PS)
+& { # create a new scope
+  $_task_path = [Text.Encoding]::UTF8.GetString(
```

Comment:
To support PS2, I believe `[Text.Encoding]` and `[Convert]` need to have the `System.` prefix, like `[System.Text.Encoding]` and `[System.Convert]`.  Since we have the JSON polyfill here to support PS2 (it's not necessary in PS3 which includes`ConvertTo-Json` already) that means we also need to make sure other PS2 compatibility fixes are in place.  PS2 testing is currently a blind spot.

We ran into this while building the PowerShell module. /cc @jpogran

Reply:
ok